### PR TITLE
Project nits, renames, readme, etc

### PR DESCRIPTION
**Pull request by felipecsl**
😄

### CODE REVIEW

**Comment by felipecsl** [comment]
@lopopolo you should enable CI for PRs ;)

File: .gitignore [hunk 1]
```diff
@@ -70,6 +70,9 @@ typings/
 # parcel-bundler cache (https://parceljs.org/)
 .cache
 
+# IntelliJ config dir
+.idea/
```
Comment (lopopolo) [inline .gitignore]: Can you regenerate this gitignore using the gitignore.io link at the top and add vim and IntelliJ?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ with [Rocket](https://rocket.rs/).
 _ferrocarril_ means _railway_ in Spanish and sounds like _ferrous_ which means
 _containing iron_.
 
+## Usage
```
Comment (lopopolo) [inline README.md]: foolsgold is one of several binary crates in this workspace. It is mostly a proof of concept/demo. Can we scope this usage under the `## foolsgold` header as an h3?

File: foolsgold/static/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>
-      FoolsGold Ruby Rack in Rust Server Extravaganza
+      Ferrocarril Ruby Rack in Rust Server Extravaganza
```
Comment (lopopolo) [inline foolsgold/static/index.html]: Ditto ferrocarril is the workspace and eventual rails demo. Fools gold is a separate proof of concept.

File: README.md [hunk 4]
```diff
@@ -9,6 +9,14 @@ with [Rocket](https://rocket.rs/).
...
+
+```bash
+cargo run --bin foolsgold
```
Comment (lopopolo) [inline README.md]: Thank you for adding this. Would you be up for adding the same thing for the `rirb` binary from the `mruby-bin` crate?

Comment (felipecsl) [reply README.md]: Yes but I'm not sure what we want me to add here exactly 😅

Comment (lopopolo) [reply README.md]: Filed GH-35

File: .gitignore [hunk 5]
```diff
@@ -160,11 +242,23 @@ Cargo.lock
 *.sass.map
 *.scss.map
 
-# End of https://www.gitignore.io/api/node,sass,rust,ruby
-
-# Overrides
```
Comment (lopopolo) [inline .gitignore]: Oops these are manual overrides that need to be preserved! Sorry for missing this.

**Comment by lopopolo** [comment]
Thanks @felipecsl!

File: README.md (hunk 4 above)
Comment (lopopolo) [reply README.md]: this got fixed in GH-50 💪

**Pull request merged**